tests/Earthfile

```diff
@@ -171,6 +169,7 @@ ga-no-qemu-group4:
     BUILD --pass-args ./autoskip+test-all
     BUILD +if-exists
     BUILD +let-set
+    BUILD +let-scope
```

Comment:
this BUILD was accidentally missed when the target was added, fortunately it's still passing :sweat_smile: